modules/simple_forms_api/app/controllers/simple_forms_api/v1/callbacks_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module SimpleFormsApi
+  module V1
+    class CallbacksController < ApplicationController
+      include ActionController::HttpAuthentication::Token::ControllerMethods
+
+      skip_before_action :verify_authenticity_token, only: [:create]
```

Comment:
The `NodCallbacksController` skips `:verify_authenticity_token`. Can we dismiss this or is it important?

Reply:
What is hitting this callback action? IMO it's probably not feasible or practical to require a third party requester  to provide a CSRF token for a callback action

Reply:
VANotify will hit this endpoint so yeah I guess it's ok to leave as-is, skipped?

Reply:
> IMO it's probably not feasible or practical to require a third party requester to provide a CSRF token for a callback action

Agreed, if we're concerned about security with only this authentication approach, perhaps we could go with additional authentication by using an HMAC, where the third-party (VANotify, in this instance) signs the payload with the HMAC, and then we check that signature ('X-Signature') against a shared secret key. But, I imagine that would require some additional setup with ensuring the shared secret is generated and stored. I'm not sure if the tradeoff is there. My two-cents is this authentication method is valid for now, but I'd defer to Platform's direction.

Reply:
We don't have any hard set guidance on this unfortunately. I do see other places throughout the code that disable the CSRF check. I think the HMAC solution offers caution and practicality but I guess we can also say that VANotify is a trusted service within the network and could maybe reduce the need for the extra layers of security here. @kenmayo Would you mind providing some security guidance here please?

Reply:
I think since we're doing this in other places AND it's a trusted service, there could be a man in the middle, but not super likely. Also, if this goes through the fwdproxy in any way, there is mTLS enabled between fwdproxy and external services. I will let you all decide on the HMAC integration